doc/item_db.txt

```diff
@@ -159,10 +159,11 @@ Baby        - Baby classes (no Third-Baby classes).
 Third       - Third classes (no Transcedent-Third or Third-Baby classes).
 Third_Upper - Transcedent-Third classes.
 Third_Baby  - Third-Baby classes.
+Fourth      - Fourth classes.
 All_Upper   - All Transcedent classes
 All_Baby    - All baby classes
 All_Third   - Applies to all Third classes.
-Fourth      - Fourth classes.
+All_Fourth  - Applies to all Fourth classes.
```

Comment:
Unnecessary for now there is only Fourth class, this field is in case of baby 4th for example ?

Another note, the field is missing in yaml2sql converter

Reply:
Removed for the time being - as discussed in Discord